config/sync/google_tag.settings.yml

```diff
@@ -0,0 +1,28 @@
+uri: 'public:/'
```

Comment:
Shouldn't this be `public://`?

Reply:
I was a little confused by that too. but this is the default settings. i'm not 100% sure why its only 1 slash